src/functionalTest/java/uk/gov/hmcts/opal/steps/draftaccount/DraftAccountPutSteps.java

```diff
@@ -92,9 +92,262 @@ public void checkCreatedAtTime() {
 
         assertEquals(createdAtTime, createdAt, "Created at time has changed");
 
+        Serenity.recordReportData().withTitle("Times").andContents(
+            "Created at time: " + createdAtTime + "\nResponse created at time: " + createdAt);
+    }
+
+    @When("I attempt to put a draft account with an invalid token")
+    public void putADraftAccountWithAnInvalidToken() {
+        SerenityRest
+            .given()
+            .header("Authorization", "Bearer " + "invalidToken")
+            .accept("*/*")
+            .contentType("application/json")
+            .when()
+            .get(getTestUrl() + DRAFT_ACCOUNT_URI);
+    }
```

Comment:
you're using the method for a get, not a put here.

Reply:
Pushed the changes